lib/model/folder.go

```diff
@@ -641,7 +627,6 @@ func (f *folder) monitorWatch(ctx context.Context) {
 				if _, ok := err.(*fs.ErrWatchEventOutsideRoot); ok {
 					l.Warnln(err)
 					warnedOutside = true
-					return
```

Comment:
Returning here is clearly wrong: We still need to stop the watch, and retry (otherwise what's the point of `warnedOutside`). The bug is still minor as it only comes up when the watcher reports some event outside of the folder.